src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,44 @@
+package study;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void size() {
+        assertThat(3).isEqualTo(numbers.size());
+    }
```

Comment:
다음과 같이 해보면 어떨까요? 😄 

assertThat( 실제 값 ).isEqualTo( 예상되는 값 );

<img width="541" alt="스크린샷 2023-04-06 오후 10 19 14" src="https://user-images.githubusercontent.com/48339310/230390544-60181847-cb3d-4645-a226-003cc7f80e62.png">

<img width="408" alt="스크린샷 2023-04-06 오후 10 20 36" src="https://user-images.githubusercontent.com/48339310/230390900-9208abbb-e482-411d-8928-16204b436a2c.png">

```suggestion
    @Test
    void size() {
        assertThat(numbers.size()).isEqualTo(3);
    }
```